pushapkscript/script.py

```diff
@@ -15,9 +15,10 @@
 
 from pushapkscript import jarsigner
 from pushapkscript.apk import sort_and_check_apks_per_architectures
+from pushapkscript.googleplay import publish_to_googleplay, should_commit_transaction, \
```

Comment:
Tiny nit: I find using parenthesis slightly cleaner than with slashes but it's totally a matter of personal options.

Reply:
Yeah, I'm fine with both, to be honest. I actually don't know if there's a PEP that defines what's the best usage of each solution